# DP: Don't add /usr/local/include for cross compilers. Assume that
# DP: /usr/include is ready for multiarch, but not /usr/local/include.

--- a/src/gcc/cppdefault.c
+++ b/src/gcc/cppdefault.c
@@ -66,8 +66,11 @@
 #ifdef LOCAL_INCLUDE_DIR
     /* /usr/local/include comes before the fixincluded header files.  */
     { LOCAL_INCLUDE_DIR, 0, 0, 1, 1, 2 },
+#if 0
+    /* Unsafe to assume that /usr/local/include is ready for multiarch.  */
     { LOCAL_INCLUDE_DIR, 0, 0, 1, 1, 0 },
 #endif
+#endif
 #ifdef PREFIX_INCLUDE_DIR
     { PREFIX_INCLUDE_DIR, 0, 0, 1, 0, 0 },
 #endif
